src/Build/NuGet.Build.Packaging.Tasks/Extensions.cs

```diff
@@ -34,6 +34,15 @@ public static string GetNullableMetadata(this ITaskItem taskItem, string metadat
 			return value;
 		}
 
+		public static string GetContentFileInclude(this ITaskItem taskItem)
```

Comment:
this is mixed in in this PR, it belongs to a different one that likely won't get merged soon. It would be best to separate it.